lib/l10n/intl_en.arb

```
@@ -1144,5 +1144,20 @@
+  "deleteLogsTooltip": "Delete logs",
+  "logsDeletedMessage": "🧹 Logs cleared",
+  "shareLogsTooltip": "Share logs file",
+  "logsShareText": "MostroApp Logs",
+  "errorSharingLogs": "An error occurred while sharing logs.",
+  "noLogsMessage": "No logs yet.",
+  "logsMenuTitle": "View Logs",
+  "logsMenuSubtitle": "Internal diagnostics and events",
+  "logsEnabledDescription": "Saving activity logs",
+  "logsDisabledDescription": "Logs disabled",
+  "viewLogsButton": "View logs",
+  "errorSharingLogs": "Error sharing logs"
```

RuboCop (1.81.6)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)